pkg/webhook/server/server.go

```diff
@@ -85,7 +85,6 @@ func (s *WebhookServer) Start() error {
 		err := srv.ServeTLS(listener, "", "")
 		if err != nil {
 			log.Errorf("Error starting Webhook https server: %v", err)
-			//os.Exit(1)
```

Comment:
What will happen if the server crashes? I think it is better to restart the shell-operator in this case. There are two ways to do it:
1. Shutdown the shell-operator if one of the webhooks crashes.
2. Provide some sort of health check endpoint to ensure that everything works.

Reply:
Ouch, I need to return this back.

Reply:
Reverted!